src/CommandingOfficers/Commander.java

```diff
@@ -567,4 +530,23 @@ public int getIncomePerTurn()
     }
     return count * (gameRules.incomePerCity + incomeAdjustment);
   }
+
+  private final ArrayList<UnitModifier> unitMods;
+  @Override
+  public List<UnitModifier> getModifiers()
+  {
+    // TODO Add call to pull modifiers from Army when that becomes a thing?
+    return new ArrayList<UnitModifier>(unitMods);
+  }
+
+  @Override
+  public void add(UnitModifier unitModifier)
+  {
+    unitMods.add(unitModifier);
+  }
+  @Override
+  public void remove(UnitModifier unitModifier)
```

Comment:
`add` and `remove` are odd function names for a `Commander` object.

Reply:
Some of that will get pulled up to `Army`, but yeah, `Commander` is the primary extension point in the engine so it needs its fingers in a lot of pies.